__tests__/app.test.js

```diff
@@ -75,3 +75,26 @@ describe("task 3. GET /api", () => {
       });
   });
 });
+
+describe("/api/articles", () => {
+  test("GET 200. should return an array of all articles, sorted by DESC (earliest if the first), has no body as property", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(articleData.length);
+        expect(articles[0].article_id).toBe(3);
```

Comment:
We need to check that all the articles are sorted. We can do this with something such as jest-sorted